### PR TITLE
Comments in README.md

### PR DESCRIPTION
**Pull request by mfloresruiz**
Changes to specify the first git commands.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,6 @@ Git collaboration workshop
 - 'git switch -c <branch_name>': create and move in 1 command
 	'git checkout -b <branch_name>'
 
+This is text that I haven't commited yet.
+
```
Comment (mfloresdeh) [inline README.md:19]: This seems to be uncompleted.  Please review.

Comment (mfloresdeh) [reply README.md:19]: @mfloresruiz please check this

**Comment by mfloresdeh** [comment]
Please review the code messages

**Comment by mfloresruiz** [comment]
Sorry I can't see any code message

image

File: README.md (hunk 1 above)
Comment (mfloresdeh) [reply README.md:19]: Not solved

**Pull request merged**